buildkite/setup-docker.sh

```diff
@@ -163,11 +163,11 @@ EOF
 
 ### Install Swift toolchains.
 {
-  curl -fsSL https://mirror.bazel.build/swift.org/builds/swift-4.2.1-release/ubuntu1404/swift-4.2.1-RELEASE/swift-4.2.1-RELEASE-ubuntu14.04.tar.gz | \
+  curl -fsSL https://mirror.bazel.build/swift.org/builds/swift-5.2.5-release/ubuntu1604/swift-5.2.5-RELEASE/swift-5.2.5-RELEASE-ubuntu16.04.tar.gz | \
       tar xz -C /opt
-  curl -fsSL https://mirror.bazel.build/swift.org/builds/swift-4.2.1-release/ubuntu1604/swift-4.2.1-RELEASE/swift-4.2.1-RELEASE-ubuntu16.04.tar.gz | \
+  curl -fsSL https://mirror.bazel.build/swift.org/builds/swift-5.2.5-release/ubuntu1804/swift-5.2.5-RELEASE/swift-5.2.5-RELEASE-ubuntu18.04.tar.gz | \
       tar xz -C /opt
-  curl -fsSL https://mirror.bazel.build/swift.org/builds/swift-4.2.1-release/ubuntu1804/swift-4.2.1-RELEASE/swift-4.2.1-RELEASE-ubuntu18.04.tar.gz | \
+  curl -fsSL https://mirror.bazel.build/swift.org/builds/swift-5.2.5-release/ubuntu2004/swift-5.2.5-RELEASE/swift-5.2.5-RELEASE-ubuntu20.04.tar.gz | \
```

Comment:
Can someone help get these uploaded?

Reply:
Done!

Reply:
Thanks! What's the next step to get this merged and these images updated?

Reply:
In theory we'd merge this PR and deploy new docker images. However, I was warned that this change will likely break some of our tests. Let's give it a try..

Reply:
Also, swift on our MacOS machines seems to be 5.1. Is this enough, or should we update to 5.2.5, too?

Reply:
Swift on macOS is part of Xcode, I think, so its version is automatically upgraded with Xcode.

Reply:
Yep this is correct ^

It would be fine with me if we also changed the default Xcode version to 11.5, which would then match this. But also it seems like that can be done in each repos config which is fine for me as a fallback